sitepod-cli.php

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/php -q
+<?php
+/*
+ * This file is part of Sitepod.
+ *
+ * Sitepod is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Sitepod is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Sitepod. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+// TODO: add verbose text, html, and quiet text flags
+$text_only = 1;
+
+ob_start (); // Start buffering output
```

Comment:
I would suggest to avoid comments like this.
The code should explain itself.
ob_start() is basic function, we don't need to explain it.
ob_get_clean() and strip_tags() are also basic functions.